app/src/main/java/edu/cascadia/mobas/campusguidebook/data/SampleData.java

```diff
@@ -24,15 +24,27 @@
     public static final Event[] events = {
             new Event("Math Club - Weekly Meeting", "Meets every other Tuesday at 3:30", "CC1-210", ZonedDateTime.of(2022,3,1,15,30,0,0, zoneId)),
             new Event("Engineering Club - Symposium", "The biggest Engineering event of the year", "CC2-120", ZonedDateTime.of(2022, 10, 7, 11, 0, 0, 0, zoneId)),
+            new Event("Science Club - Weekly Meeting", "Meets every other Tuesday at 3:30", "CC1-210", ZonedDateTime.of(2022,3,1,15,30,0,0, zoneId)),
+            new Event("Art Club - Symposium", "The biggest Art event of the year", "CC2-120", ZonedDateTime.of(2022, 10, 7, 11, 0, 0, 0, zoneId)),
+            new Event("Programming Club - Weekly Meeting", "Meets every other Tuesday at 3:30", "CC1-210", ZonedDateTime.of(2022,3,1,15,30,0,0, zoneId)),
+            new Event("Robotics Club - Symposium", "The biggest Robotics event of the year", "CC2-120", ZonedDateTime.of(2022, 10, 7, 11, 0, 0, 0, zoneId)),
     };
 
     public static final Club[] clubs = {
             new Club("Math Club", "A group for students who love math or who would like to learn more", "Mathy McMathface <mathy@mathface.com", "Almathia Newton <anewton@gmail.com>"),
             new Club("Engineering Club", "A group for students who love to design, fabricate, and build", "Howard Wolowitz <hwolowitz@caltech.edu", "Motor Close <motorclose@badjoke.com>"),
+            new Club("Science Club", "A group for students who love science or who would like to learn more", "Mathy McMathface <mathy@mathface.com", "Almathia Newton <anewton@gmail.com>"),
+            new Club("Art Club", "A group for students who love to paint, sculpture, and design", "Howard Wolowitz <hwolowitz@caltech.edu", "Motor Close <motorclose@badjoke.com>"),
+            new Club("Programming Club", "A group for students who love coding or who would like to learn more", "Mathy McMathface <mathy@mathface.com", "Almathia Newton <anewton@gmail.com>"),
+            new Club("Robotics Club", "A group for students who love to  design, construction, operation, and use of robots.", "Howard Wolowitz <hwolowitz@caltech.edu", "Motor Close <motorclose@badjoke.com>"),
```

Comment:
Some Emails are missing the closing diamond bracket > (e.g. \"<hwolowitz@caltech.edu") but this won't affect performance.  Thanks for this.. We really needed more sample data.